tensorflow_addons/custom_ops/image/BUILD

```diff
@@ -20,10 +20,31 @@ cc_binary(
 
 
 py_library(
-    name = "images_ops_py",
+    name = "image_py",
     srcs = ([
         "__init__.py",
         "python/__init__.py",
```

Comment:
Let's put all files in images_ops_py at first, and split them if necessary in the future. So just append `python/dense_image_warp.py` here ?

Reply:
Sure. My previous concern here was because of the naming consistence [here](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/custom_ops/BUILD). Seems that the text part does not suffix `_ops`.

Reply:
Yeah, good point. How about `image_ops_library` and `text_ops_library`? Because those modules contain both python and c++ codes. cc @seanpmorgan what do you think?

Reply:
Sounds good to me. So all files should be in `image_ops_py` and the `image_ops_library` in the upper BUILD file should depend on `image_ops_py`, right?

Reply:
Let's rename `image_ops_py` to `image_ops`, and add new file in its `srcs` block. I think it is enough. What do you think?

Reply:
It would be a straight solution if all new files are in `image_ops`'s `srcs` block.

But when it comes to `py_test`, should we make it depend on `image_ops` (put it on `deps` in `py_test` like what `transform_ops_test` does now)? If not, there will be lots of redundant codes for its `srcs` and `data` (for `*.so`) blocks. If yes, I think it's not a **unit test** because tests will execute after compiling other codes instead of its own dependence.

Therefore, I think it would be better if we can separate each operation (say `transform_ops` and `dense_image_warp_ops` here) into different `py_library`. And both the upper wrapper `image_ops` and each `py_test` can directly depend on it without too many duplicated lines.

Reply:
I agree, and I understand. Just for simplicity, and easy to maintain. Let's put all src together at first, then we can split them into groups if necessary in the future.

Reply:
A trade off here. And I think we can leave a TODO comment here to remind us. Will you like the solution?

Reply:
Yeah, it's okay for me 😄